dali/pipeline/operator/op_spec.h

```diff
@@ -566,7 +543,10 @@ inline std::vector<T> OpSpec::GetRepeatedArgumentImpl(const string &name) const
   } else {
     // Argument wasn't present locally, get the default from the associated schema
     const OpSchema& schema = GetSchema();
-    return detail::convert_vector<T>(schema.GetDefaultValueForOptionalArgument<V>(name));
+    DALI_ENFORCE(
+        schema.HasArgumentDefaultValue(name),
+        make_string("Optional argument \"", name, "\" has no default value and was not set."));
+    return detail::convert_vector<T>(schema.GetDefaultValueForArgument<V>(name));
```

Comment:
Likewise - why not let the GetDefaultValue throw?

Reply:
Done